samples/javascript_nodejs/21.corebot-app-insights/bots/dialogAndWelcomeBot.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { CardFactory } = require('botbuilder');
+const { DialogBot } = require('./dialogBot');
+const WelcomeCard = require('./resources/welcomeCard.json');
+
+class DialogAndWelcomeBot extends DialogBot {
+    /**
+     * 
```

Comment:
Can we remove this space? Same on `dialogbot.js` file

Reply:
Feedback Applied.